src/ObjectPool/test/ThreadingTest.cs

```diff
@@ -13,10 +13,22 @@ public class ThreadingTest
         private bool _foundError;
 
         [Fact]
-        public void RunThreadingTest()
+        public void DefaultObjectPool_RunThreadingTest()
         {
-            _cts = new CancellationTokenSource();
             _pool = new DefaultObjectPool<Item>(new DefaultPooledObjectPolicy<Item>(), 10);
+            RunThreadingTest();
+        }
+
+        [Fact]
+        public void DisposableObjectPool_RunThreadingTest()
```

Comment:
`DisposableObjectPool` handles the `Return` by itself, so I think this should be tested.